### DISCUSSION TITLE
Invalid `prisma.table.deleteMany()` invocation

### DISCUSSION THREAD

**Discussion by jatingodnani** [prisma/prisma#21301] [Q&A]
### Question

i am getting this error after create a Table model Invalid `prisma.table.deleteMany()` invocation:

Operation 'deleteMany' for model 'Table' does not match any query. only getting on model table and all model are working find.can anyone tell what could be the reason
same error when i creating data
### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by nurul3101** [comment]
Hello @jatingodnani 👋 

Can you try invoking `npx prisma generate` command and check if you still get the same error?

**Comment by jatingodnani** [comment]
It is working but my one ques is when u first load the website why it is
not fetching data instantly it fetch data after after second reloading

**Comment by BenGF23** [comment]
I got this error as well and solved by running the generation again: if using prisma would be a command like: npx prisma generate, and running the migration again: npx prisma migrate...

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
